style_guides/accessibility_guide.md

```diff
@@ -50,6 +50,39 @@ You should always prefer the `<label for>` solution, since it also adds benefit
 for every user, by making the label clickable, to directly jump to the form
 element (or in case of checkboxes and radio buttons directly check them).
 
+#### How to generate ids?
+
+When labeling element (and for some other accessibility tasks) you will often need
+ids. Ids must be unique within the page (i.e. no duplicate ids in the rendered DOM
+at any time).
+
+Since we have some components, that are used multiple times on the page, you must
```

Comment:
You can remove the first comma and say `Since we have some components that are used multiple times on the page,`